### PR TITLE
Added /adventure command

### PR DESCRIPTION
**Pull request by TheWadeSmith**

### CODE REVIEW

**Comment by Wild1145** [comment]
Not sure what point it would serve in game but I guess someone somewhere has asked for it...

**Comment by JeromSar** [comment]
@CrafterSmith12 I don't understand why players would like to be in adventure mode. Please enlighten me on that. :)

**Comment by CrafterSmith12** [comment]
@JeromSar They may want to role-play or pvp.

File: src/me/StevenLawson/TotalFreedomMod/Commands/Command_adventure.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+import me.StevenLawson.TotalFreedomMod.TFM_AdminList;
+import me.StevenLawson.TotalFreedomMod.TFM_Util;
+import org.bukkit.GameMode;
+import org.bukkit.command.Command;
+import org.bukkit.command.CommandSender;
+import org.bukkit.entity.Player;
+
+@CommandPermissions(level = AdminLevel.OP, source = SourceType.BOTH)
+@CommandParameters(description = "Quickly change your own gamemode to adventure, or define someone's username to change theirs.", usage = "/<command> [partialname]", aliases = "gmsp")
```
Comment (JeromSar) [inline src/me/StevenLawson/TotalFreedomMod/Commands/Command_adventure.java]: Incorrect alias here. :)

**Comment by CrafterSmith12** [comment]
@JeromSar

**Comment by JeromSar** [comment]
@CrafterSmith12 Thanks for contributing, this PR will be merged soon.

**Comment by Wild1145** [comment]
@JeromSar Re-Open?

**Comment by DUFCLiam** [comment]
Should be closed due to the release of 5.0; @JeromSar

**Review by Wild1145 — APPROVED** [review]
Changes are ready to merge.

**Comment by mayokek** [comment]
@Wild1145, this is made for TFM 4.3 not 5.0.

**Comment by Wild1145** [comment]
@AvalancheYT @JeromSar Manually merges PR's, there is every chance that he would be able to merge this.

**Comment by JeromSar** [comment]
@Wild1145 is right. :)

**Comment by ghost** [comment]
Why hasn't this merged yet?

**Comment by Wild1145** [comment]
@Pramire I am not sure, it is ready to be merged in and im sure will be merged in when @JeromSar is working towards the next release.

**Comment by JeromSar** [comment]
This was added in https://github.com/TotalFreedom/TotalFreedomMod/commit/baf73df4f9be81f0cb39b71e6e1809425db649b8

**Pull request closed without merging**